docs/source/prodflow.rst

```diff
@@ -23,16 +24,41 @@ Data
 ------------
 Data collected in NEXT detectors is in form of *Raw Waveforms* (**RWF**, in ADCs). The following production flow is used over the **RWFs** for both type of Triggers:
 
-  .. image:: images/prodflow_data.png
+  .. image:: images/prodflow_data2.png
```

Comment:
Can we rename it to `prodflow_data_v2.0` to match the version of IC that implements this flow?

Reply:
yes, sounds good to me